ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/helpcenter/articlepage/ArticleContainerFragmentTest.java

```diff
@@ -0,0 +1,198 @@
+package com.kayako.sdk.android.k5.helpcenter.articlepage;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.support.v7.widget.Toolbar;
+import com.kayako.sdk.android.k5.R;
+import com.kayako.sdk.android.k5.common.fragments.BaseContainerFragment;
+import com.kayako.sdk.helpcenter.articles.Article;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.spy;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+        ArticleContainerFragment.class,
+        ArticleContainerFactory.class,
+        Bundle.class,
+        Fragment.class,
+        ArticleFragment.class,
+        BaseContainerFragment.class
+})
+public class ArticleContainerFragmentTest {
+    private static final String ARG_ARTICLE = "article";
+    private ArticleContainerFragment articleContainerFragment;
+
+    @Mock
+    private Article article;
+
+    @Mock
+    private ArticleContainerContract.Presenter presenter;
+
+    @Mock
+    private Bundle bundle;
+
+    @Mock
+    private View view;
+
+    @Mock
+    private LayoutInflater layoutInflater;
+
+    @Mock
+    private ViewGroup viewGroup;
+
+    @Mock
+    private Toolbar toolbar;
+
+    @Mock
+    private AppCompatActivity fragmentActivity;
+
+    @Mock
+    private ActionBar actionBar;
+
+    @Mock
+    private ArticleFragment articleFragment;
+
+    @Mock
+    private FragmentManager fragmentManager;
+
+    @Mock
+    private FragmentTransaction fragmentTransaction;
+
+    @Mock
+    private Menu menu;
+
+    @Mock
+    private MenuInflater menuInflater;
+
+    @Mock
+    private MenuItem menuItem;
+
+    @Test
+    public void givenArticleWhenNewInstanceThenReturnFragment()
```

Comment:
✅ Good to see tests are being added, therefore, increasing the overall quality of the project.

**No change suggested, looks good!**	  

<a id="6e186af0-4e59-11e8-860c-69c5dc5809f4" href="none" codereview-uuid-reserved-tag />